frontend/src/app/(routes)/staking/components/StakingCard.tsx

```diff
@@ -213,18 +221,24 @@ const StakingCardActions = ({
           <StakingCardActionButton
             name={'Delegate'}
             action={delegate}
-            txStatus={TxStatus.IDLE}
+            isPending={false}
```

Comment:
"Is it okay to set 'isPending' as false and make it static?"

Reply:
It was like that before this pr.. ( TxStatus.IDLE === TxStatus.Pending always false)

The loading for delegate action is shown in the popup itself, not in the card.